content/en/error_tracking/custom_grouping.md

```diff
@@ -81,17 +77,128 @@ logger.error('Error processing request', extra={'error.fingerprint': 'my-custom-
 In this case, `my-custom-grouping-material` is used to group these error logs into a single
 issue in Error Tracking.
 
-[1]: /logs/log_collection/
+#### Mobile Example
+
+In Datadog's mobile SDKs, you can add a custom error fingerprint when logging an error by adding
+a predefined attribute to the log call:
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+In order to use custom grouping, you need the Datadog iOS SDK `2.8.1` or higher.
+
+```swift
+let errorFingerprint = "my-custom-grouping-material"
+logger.error(
+  "My error message",
+  error: error,
+  attributes: [
+    Logs.Attributes.errorFingerprint: errorFingerprint
+  ]
+)
+```
+{{% /tab %}}
+
+{{% tab "Android" %}}
+In order to use custom grouping, you need the Datadog Android SDK `2.7.0` or higher.
+
+```kotlin
+val errorFingerprint = "my-custom-grouping-material"
+val attributes = mapOf(LogAttributes.ERROR_FINGERPRINT to errorFingerprint)
+logger.e("My error message", error, attributes)
+```
+{{% /tab %}}
+
+{{% tab "React Native" %}}
+TODO:
+{{% /tab %}}
+
+{{% tab "Flutter" %}}
+In order to use custom grouping, you need the Datadog Flutter SDK `2.4.0` or higher.
+
+```dart
+final errorFingerprint = "my-custom-grouping-material";
+logger.error(
+  'My error message',
+  errorStackTrace: st,
+  attributes {
+    DatadogAttributes.errorFingerprint: "my-custom-grouping-material",
+  }
+);
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+Or, you can add or adjust the fingerprint in the log mapper:
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+In order to use custom grouping, you need the Datadog iOS SDK `2.8.1` or higher.
+
+```swift
+let logsConfiguration = Logs.Configuration(
+  eventMapper: {
+      log.error?.fingerprint = "my-custom-grouping-material"
+      return log
+  }
```

Comment:
I think it won't compile because `log` is unknown in this context. Also it must be mutable, so:
```suggestion
  eventMapper: { log in
      var log = log
      log.error?.fingerprint = "my-custom-grouping-material"
      return log
  }
```